validation_server/validation_server.js

```diff
@@ -0,0 +1,54 @@
+const axios = require("axios");
+const { B3Propagator } = require("@opentelemetry/core");
+const { HttpTraceContext } = require("@opentelemetry/core");
+const { NodeTracerProvider } = require("@opentelemetry/node");
```

Comment:
Nothing here is node specific. I would use the basic tracer provider provided in the `tracing` package and call propagation APIs manually.